std/internal/math/biguintcore.d

```diff
@@ -673,7 +673,6 @@ public:
             uint [] wasteful = new BigDigit[x.data.length];
             wasteful[] = x.data[];
             uint rem = multibyteDivAssign(wasteful, y, 0);
-            delete wasteful;
```

Comment:
Here please use `malloc` and `free`, it'll be faster and better. Can place `free` in a `@trusted` lambda.